docs/pages/use-cases/nftqueries/nft6.mdx

```diff
@@ -0,0 +1,32 @@
+import { Callout, Code, Tabs, Steps } from "nextra/components"
+
+## 6. Buying an NFT
+
+Buying an NFT on a marketplace (often referred to as placing a "bid," where you offer ADA in exchange for assets) involves the use of smart contracts. These contracts facilitate secure agreements between parties who may not necessarily trust each other.
+
+For this exercise, we'll assume you're already familiar with the concept of smart contracts and understand how they operate. If not, consider reviewing resources such as the [Cardano Developer Portal](https://developers.cardano.org/docs/smart-contracts/) and the Aiken [EUTxO crash course](https://aiken-lang.org/fundamentals/eutxo) to name but a few. 
+
+### Steps to Buy an NFT
+The first step in buying an NFT is to select the one you want to make an offer on. To do this, refer to the “View Assets for Sale” query to see all assets listed on the marketplace.
+
+After choosing the NFT, a similar process occurs as when you are selling an NFT (refer to the previous “Selling an NFT" example for details).
+
+On the Marketplace application, when a user clicks on "Buy", the application backend will build a transaction that matches the requirements of the smart contract. This includes:
+
+1. Setting the datum with details like amount of ada charged by the asset, royalties for the artist, and marketplace service fees. This steps are necessary to unlock the NFT in the transaction  making sure that all required payouts are fulfilled
+
+2. Including the redeemer code, in our example “accept” (accept the offer/listing requirements).
+
+3. Providing the necessary transaction UTXOs and address, inputs and outputs.
+
+To successfully build the transaction, you'll need the following data, typically stored in the Ledger Sync database:
+
+- **Protocol Parameters**: Define blockchain settings necessary for calculating transaction fees and other critical parameters.
+
+- **UTXOs**: The Unspent Transaction Outputs associated with the wallet that will fund the transaction.
+
+- **Slot/Block Tip**: The current slot or block tip to ensure the transaction is valid within the blockchain context.
+
```

Comment:
Maybe here we can point the user to the example which we have the mentioned queries?